### ISSUE TITLE
Color in UI Page - Palettes Only

### ISSUE THREAD

**Issue by shinytoyrobots** [carbon-design-system/carbon-website#95]
@jeanservaas or @jeoffw to populate this issue

**Comment by claycrenshaw** [comment]
@jeanservaas @jeoffw @aagonzales @chrisconnors-ibm Please find the latest revisions for feedback here: http://typical-use.surge.sh/design/product/experimental/color

You can leave your feedback right here in the issue. Thanks!

Notes: There was some discussion around hiding or altering the interactive and hover colors. Please advise on this.

**Comment by claycrenshaw** [comment]
I'm also attaching the latest version of this page in case anyone needs to move on it while I'm out this week.
[color.zip](https://github.com/carbon-design-system/carbon-website/files/2457881/color.zip)

**Comment by jeanservaas** [comment]
**General Intro**
I think this page could be a little more inviting, that’s why I included that high-level visual (animated gif) of the examples of color in product… we should start (like easing into a hot tub) with a high-level overview of thoughts and visuals on color and then start to break it apart. (See macdown file)
![image](https://user-images.githubusercontent.com/43144260/46634630-13b33c80-cb17-11e8-8254-570713ab5b85.png)


**Color Palette**
We were talking about keeping the copy-paste color palette in a different tab so that it’s easy to bookmark it easier and we can direct link to it more easily too. Its supposed to be a quick reference and right now it feels a little buried in usage content.

(We also talked about having much larger swatches in the table and being able to select the hexes from there … but this would probably be more of a future state that we'd need to explore) 

Either way, discussion of tokenization, theming and color roles table seem disconnected now. And I miss the spectrum illustration from color anatomy because it shows you the ecosystem.


**Hovers**
I think we should remove the hover states from the color palette right now and from and kind of table (I’d just mention them in the text at the moment)  They’re unresolved in product right now (moving forward we may darken or lighten vs. having a different palette for hovers) and this list is not exhaustive (gray hovers are missing).


**Support tokens**
Not sure why these guys can’t be folded into the larger token table… again, these are just values attached to particular usage, they could change with theming also. Right now it looks floaty.

[color_in_ui.zip](https://github.com/carbon-design-system/carbon-website/files/2457962/color_in_ui.zip)
